### PR TITLE
dynamic rules: `path-exclude` rule fix

### PR DESCRIPTION
**Pull request by Hidanio**

### CODE REVIEW

**Review by Danil42Russia — COMMENTED** [review]
Why prefix `dinamic rules` (with an error) and not just `rules`?

**Comment by Hidanio** [comment]
> Why prefix `dinamic rules` (with an error) and not just `rules`?

Because this two different conceptions and we should not mix it. We have different logic for launch dynamic rules and embedded (I know that dynamic rules are called that way in the code and I think it’s wrong). We must forcefully separate this concept

File: src/tests/rules/rules_test.go [hunk 1]
```diff
@@ -488,3 +488,51 @@ function type_type_check(string $animal_name, int $animal_id) {
...
+          $hello = 'echo 123;';
+          eval($hello);
+          eval('echo 456;');
+        `
+	test.AddNamedFile("www/no", code)
+
+	test.RunRulesTest()
+}
+
+func TestRulePathExclude2(t *testing.T) {
```
Comment (Danil42Russia) [inline src/tests/rules/rules_test.go:514]: Don't give incremental tests. Give a normal name.

Comment (Hidanio) [reply src/tests/rules/rules_test.go]: Renamed

**Review by Danil42Russia — APPROVED** [review]

**Pull request merged**